page.codeberg.tpikonen.satellite.json

```diff
@@ -0,0 +1,45 @@
+{
+  "app-id": "page.codeberg.tpikonen.satellite",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "41",
+  "sdk": "org.gnome.Sdk",
+  "command": "satellite",
+  "rename-icon": "satellite",
+  "copy-icon": false,
+  "rename-desktop-file": "satellite.desktop",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=dri",
+    "--talk-name=org.gtk.vfs.*",
+    "--system-talk-name=org.freedesktop.ModemManager1.*",
+    "--filesystem=home"
```

Comment:
What does it use `home` for?

Reply:
For saving GPX-tracks.

Reply:
Ideally, it should use GtkNativeFileChooser to use a portal instead of direct filesystem access.

Reply:
This app does not use a file chooser, but writes tracks to a file with a name generated from a timestamp (like `track-2021-09-06_14:09:22.gpx`).

It currently only writes them to G_USER_DIRECTORY_DOCUMENTS (i.e. `~/Documents`), but I can further restrict this to a subdir like `satellite-tracks`. Then the needed filesystem permissions would be `--filesystem=xdg-documents/satellite-tracks:create` (if I'm reading the [docs](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#filesystem-permissions) correctly).

Would this be ok?

Reply:
Yeah, that sounds fine.

Reply:
Ok, I made this change on a new release and updated the build manifest.